lib/providers/env_var_provider.ex

```diff
@@ -156,7 +169,7 @@ defmodule EnvVar.Provider do
   defp get_env_value(key, config) do
     key
     |> System.get_env()
-    |> set_default(config[:default], key)
```

Comment:
This argument was not used.

Reply:
Leftover from previous implementation I think